### PR TITLE
Use ISymUnmanaged* interfaces from Microsoft.DiaSymReader

### PR DESCRIPTION
**Pull request by tmat**
Will need to coordinate with the debugger team.

### CODE REVIEW

**Comment by jaredpar** [comment]
What part needs to be coordinated?

**Comment by tmat** [comment]
The debugger needs to start using this library instead of their own impl. and give us a drop of the debug engine.

File: src/VisualStudio/Core/Def/ServicesVisualStudio.csproj [hunk 1]
```diff
@@ -719,4 +682,4 @@
     <Import Project="..\..\..\..\build\VSL.Imports.Closed.targets" />
     <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   </ImportGroup>
-</Project>
+</Project>
```
Comment (amcasey) [inline src/VisualStudio/Core/Def/ServicesVisualStudio.csproj]: Trailing newline?

**Comment by amcasey** [comment]
:+1:

**Comment by gafter** [comment]
@tmat This PR is quite old. Can you please close if obsolete?

**Comment by amcasey** [comment]
I updated this PR and merged it as 5f887b3d907cb4a0dd582e9f875861ed3320a669.

**Pull request closed without merging**